### ISSUE TITLE
PURL redirects to googlecode 404

### ISSUE THREAD

**Issue by dr-shorthair** [UKGovLD/linked-data-api#2]
http://purl.org/linked-data/api/vocab# appears to point at a googlecode repo which is now 404

**Comment by der** [comment]
Can't fix.

The repo itself was relocated to: https://github.com/UKGovLD/linked-data-api

However, the purl service went through turbulent times and a replacement is now being run by someone else. Supposedly users of the old service were to have been given new credentials to access their data but all that failed for us and we no longer have access to any of the purls under http://purl.org/linked-data - that applies to the Data Cube vocabulary as well.  

[My memory is that we had updated the purl service before it died but maybe that update failed, either way the new service is running on out of date links.]

**Comment by dr-shorthair** [comment]
Yeah - I watched the turmoil with some trepidation considering how many semantic web vocabularies use purl.org URIs. But I thought it was straightened out now? I was just able to login OK via the internet archive, though I couldn't check any purls because I can't recall any that I ever created, if I did (stretching my memory there - it would have been at least 10 years ago).

**Comment by dr-shorthair** [comment]
On here https://archive.org/services/purl/help it says 

**Claiming a PURL domain**

The PURL service is now administered by the Internet Archive. If you have any difficulty making changes to your PURLs please contact info@archive.org for assistance.

**Administering PURLs**

The PURL system is a service of the Internet Archive. To make changes to a PURL users need to have a user account with the Internet Archive.

**Comment by der** [comment]
My old login in credentials don't work and I've no way to prove I used to have access those purls. I wasn't the original creator of the root area, just had delegated authority to make changes.

**Comment by dr-shorthair** [comment]
Hmmm. Its tedious, but we should try to take care of these key URIs. 

I wouldn't have thought it would be difficult to make the case: 
1. The links are currently broken (so clearly no-one else is taking care of them)
2. Your name is all over easily findable documents that describe the Linked Data API (e.g. https://github.com/UKGovLD/linked-data-api) 

I did have to reset my password, but my account details were still on file after probably 3 years gap. 

FWIW I was prompted to create the ticket because I was contacted by a user who is trying to use an old ELDA config that I had posted. I went poking around the ELDA and linked-data sources and found these URIs still appear in the distros there. So I also put a separate ticket in for ELDA.

**Comment by der** [comment]
My account details are not on file, at least the error messages say "The email you entered isn't in the Internet Archive system."

**Comment by dr-shorthair** [comment]
Oh dear - surely not HP vintage !?
